e2e-tests/package-lock.json

```diff
@@ -1714,9 +1714,9 @@
       }
     },
     "node_modules/jpeg-js": {
-      "version": "0.4.3",
-      "resolved": "https://registry.npmjs.org/jpeg-js/-/jpeg-js-0.4.3.tgz",
-      "integrity": "sha512-ru1HWKek8octvUHFHvE5ZzQ1yAsJmIvRdGWvSoKV52XKyuyYA437QWDttXT8eZXDSbuMpHlLzPDZUPd6idIz+Q=="
+      "version": "0.4.4",
```

Comment:
Why does this `package-lock.json` have updates if we did not update the corresponding `package.json`?

Reply:
I believe it's because I modified `node_modules/playwright-core` to use version 4.4 for `jpeg-js` on line 1923

Reply:
Ah got it. So you actually should not ever edit `package-lock.json` directly since it is a generated file. Instead try updating the [@playwright/test dependency in package.json](https://github.com/NASA-AMMOS/aerie/blob/develop/e2e-tests/package.json#L10), then `npm install` to generate the new package-lock.json file. This was [already fixed in Playwright](https://github.com/microsoft/playwright/pull/14942) so updating to the latest (`1.25.2`) should do the trick.

Reply:
Thanks for the heads-up! I think I got confused by some version dates when I initially checked if I could change Playwright. I've changed the `package.json` as you've suggested.